### PR TITLE
10 expand terminology for metadata

### LINKED ISSUE #10: Expand terminology for metadata
The metadata [src/terminology/metadata.md](https://github.com/Remedy-Entertainment/USDBook/blob/main/src/terminology/metadata.md) file is a bit sparse. It would be great if there were some more hands-on examples of when metadata can be used for example.

### PR DESCRIPTION
**Pull request by kminnaert**
Expanding on the metadata subject

I have added three examples on where metadata is used within USD (layers, prims and properties). Currently draft as I would like to find a better way to prevent referring to later chapters so often.

### CODE REVIEW

File: src/terminology/metadata.md [hunk 1]
```diff
@@ -1,9 +1,68 @@
...
+### Layer Metadata
+
+```admonish example title=""
+![StageMetadata](../images/terminology/metadata_stage.png)
+```
+
+In the image above, the [PseudoRoot](https://graphics.pixar.com/usd/release/glossary.html#usdglossary-pseudoroot) at path `/` has been selected and the metadata panel in `usdview` has been highlighted. This root represents the `entry.usda` layer from [Animal Logic ALab](https://animallogic.com/alab/). `entry.usda` defines metadata that can be inspected and presents the following information to the user about itself:
+- The linear units used are `centimeters` → `metersPerUnit = 0.01`
+- The scene's up axis is `Y`
+- Two layers contribute to the scene (see [Local/Sublayer](./local_sublayer.md) later)
```
Comment (grische) [inline src/terminology/metadata.md]:
```suggestion
- Two layers contribute to the scene (more about this later in [Local/Sublayer](./local_sublayer.md)) → `subLayers = [...]`
```

File: src/terminology/metadata.md [hunk 2]
```diff
@@ -1,9 +1,68 @@
...
+## Simple Metadata Examples
+
+### Layer Metadata
+
+```admonish example title=""
+![StageMetadata](../images/terminology/metadata_stage.png)
+```
+
+In the image above, the [PseudoRoot](https://graphics.pixar.com/usd/release/glossary.html#usdglossary-pseudoroot) at path `/` has been selected and the metadata panel in `usdview` has been highlighted. This root represents the `entry.usda` layer from [Animal Logic ALab](https://animallogic.com/alab/). `entry.usda` defines metadata that can be inspected and presents the following information to the user about itself:
+- The linear units used are `centimeters` → `metersPerUnit = 0.01`
```
Comment (grische) [inline src/terminology/metadata.md]:
```suggestion
- The linear units used are `0.01` meters (i.e. centimeters) → `metersPerUnit = 0.01`
```

File: src/terminology/metadata.md [hunk 3]
```diff
@@ -1,9 +1,68 @@
...
+- The scene's up axis is `Y`
+- Two layers contribute to the scene (see [Local/Sublayer](./local_sublayer.md) later)
+- Animation is present at a certain rate per second → `startTimeCode = 1004.0`, `endTimeCode = 1057.0`, `framesPerSecond = 24.0`, `timeCodesPerSecond = 24.0`
```
Comment (grische) [inline src/terminology/metadata.md]:
```suggestion
- Animation is present between time code `1004.0` and `1057.0` with `24.0` frames and time codes per second.
  - `startTimeCode = 1004.0`
  - `endTimeCode = 1057.0`
  - `framesPerSecond = 24.0`
  - `timeCodesPerSecond = 24.0`
```
Might be worth it explaining the difference between fps and tcps later on.

File: src/terminology/metadata.md [hunk 4]
```diff
@@ -1,9 +1,68 @@
...
+
+### Property Metadata
+
+Lastly, there is property metadata. This data is meant to give fine grained information about the property itself. For example
+
+- What type it is → `typename = double3`
+- Whether it can change over time → `variability = Sdf.VariabilityVarying`
+- Is it a custom "out-of-schema" (see [Schemas](./schemas.md) later) property → `custom = false`
+
+```admonish example title=""
```
Comment (grische) [inline src/terminology/metadata.md]: This should probably be moved on top of the list

File: src/terminology/metadata.md [hunk 5]
```diff
@@ -1,9 +1,68 @@
...
+
+### Layer Metadata
+
+```admonish example title=""
+![StageMetadata](../images/terminology/metadata_stage.png)
+```
+
+In the image above, the [PseudoRoot](https://graphics.pixar.com/usd/release/glossary.html#usdglossary-pseudoroot) at path `/` has been selected and the metadata panel in `usdview` has been highlighted. This root represents the `entry.usda` layer from [Animal Logic ALab](https://animallogic.com/alab/). `entry.usda` defines metadata that can be inspected and presents the following information to the user about itself:
+- The linear units used are `centimeters` → `metersPerUnit = 0.01`
+- The scene's up axis is `Y`
```
Comment (grische) [inline src/terminology/metadata.md]:
```suggestion
- The scene's up axis is `Y` → `upAxis = Y`
```

File: src/terminology/metadata.md [hunk 6]
```diff
@@ -1,9 +1,68 @@
...
+```admonish example title=""
+![StageMetadata](../images/terminology/metadata.png)
+```
+
+Metadata on Prims can infer a lot of information about the prim itself, how it is used in the scene (aka [Composition](./composition.md)), what kind of properties it has, etc.
+
+From the above we know that the prim at `/root/alab_set01/lab_electronics01_0001/bench01/decor_paper_notej01_0001` 
+
+- Is an `Xform` → `typeName = Xform`
+- It is instanced (see [Instancing](./instancing.md) later) → `instanceable = true`
```
Comment (grische) [inline src/terminology/metadata.md]:
```suggestion
- It is instanced (more about [Instancing](./instancing.md) later) → `instanceable = true`
```

File: src/terminology/metadata.md [hunk 7]
```diff
@@ -1,9 +1,68 @@
...
+- Has two VariantSets (see [VariantSet](./variantset.md) later) → `geo` and `geo_vis`
```
Comment (grische) [inline src/terminology/metadata.md]:
```suggestion
- Has two VariantSets (more about [VariantSets](./variantset.md) later) → `geo` and `geo_vis`
```

File: src/terminology/metadata.md [hunk 8]
```diff
@@ -1,9 +1,68 @@
...
+- Is categorized as a `component` → `kind = component`
```
Comment (grische) [inline src/terminology/metadata.md]:
```suggestion
- Is categorized as a `component` (more about the [Kind](./kind.md) metadatum later) → `kind = component`
```

File: src/terminology/metadata.md [hunk 9]
```diff
@@ -1,9 +1,68 @@
...
+- Etc.
+
+```admonish abstract title=""
+See the [Kind](./kind.md) chapter for an elaborate example on the `kind` metadatum can be used.
```
Comment (grische) [inline src/terminology/metadata.md]: I think it is a bit too early for this note. The Kind metadata page will come only a few chapters down the road.

**Review by grische — COMMENTED** [review]
I really like the new expanded metadata chapter, thanks a lot!

I added a few suggestions for more consistency and making it easier to read.

File: src/terminology/metadata.md (hunk 3 above)
Comment (kminnaert) [reply src/terminology/metadata.md]: Agreed, although I'm not certain yet where to put that information. Usd is full of these kind of minutiae that are arguably less on the terminology side of things, but more on "workflow". There has always been the idea to expand on the book by adding a new top level chapter focusing on workflows and I can see this fitting in there.

File: src/terminology/metadata.md (hunk 9 above)
Comment (kminnaert) [reply src/terminology/metadata.md]: Fair point, I'll remove it.

**Review by grische — APPROVED** [review]
I think the CI needs a nudge, otherwise LGTM. Thanks!

**Comment by kminnaert** [comment]
Yeah, not sure what's going on there but I'll investigate ASAP. Probably some version mismatching.

**Pull request merged**